### ISSUE TITLE
Reorder pages in the Payments section

### ISSUE THREAD

**Issue by GBKS** [BitcoinDesign/Guide#596]
This is a summary of several conversations (here's [one of them](https://github.com/BitcoinDesign/Guide/pull/589#discussion_r768287811)). Putting this in a separate PR avoids merge conflicts and allows us to tackle this in one sweep instead of having to revisit as PRs get merged.

- Move "Payment request formats" out of the sub-sub-folder and make it a regular page of the section, as it is relevant to multiple pages
- Move "Coin selection" (location TBD, either "Payments" top-level, or "Glossary")
- Move "Transactions" further back in the section as its fairly technical (start with the UX, then go into technical details)
- Move "Receiving bitcoin" before "Sending bitcoin" as that might be the more natural order

We can finalize whether we want to do all these steps, only some of them, etc.

**Comment by Bosch-0** [comment]
Agree with the structure you've laid out here @GBKS. Happy to tackle this once https://github.com/BitcoinDesign/Guide/pull/611 is merged.
